reference-sheets/GitPractices.md

```diff
@@ -0,0 +1,204 @@
+
+There is another way to merge main into your code called **Rebasing**, but it's complicated to explain in writing and you'll never use it with a team as small as a high school robotics team. The idea behind it is that you rewrite your commit history during a rebase, and that **instead** of branching off of the main branch that was four commits ago, you are **now** technically branched off of the newest main commit. It's useful for organizing commits to turn the visualization from a web to a tree, but when you're doing such small changes to a tiny codebase like used in robotics, it doesn't even matter. 
+
+# Usage
+
+Looks like you got through all the conceptualization. Nice job! However, that means absolutely nothing if you don't know how to put it into practice. So, in this section, we will go over the actual git command line inputs.
+
+Opening the command line isn't a difficult task. Visual Studio Code has one built in (accessible by pressing ctrl+\`), which makes using git much easier. 
+If you're not using VSCode, 
+Windows: Press *Win + X* to open the Command Prompt.
+Mac: Press *Cmd + Space* to open spotlight search, then type "Terminal" to open the Terminal.
+Linux: Spin around five times and then sing Mary Had A Little Lamb.
+
+### Cloning
+
+There are a few ways to clone a Github repository onto your computer, but the best way and the way that we'll be going over is by using the command line. 
+
+Go onto the GitHub website, then find the repository. There should be a green button that says "Code". Click that, and then copy the link that shows up under it. This will be used in the command. It tells git what exactly to clone off of the internet.
+
+Next, navigate using the command line to the target folder where you want your cloned repository to be on your computer. You can alternatively move the cloned folder after your clone it, but moving around using the command line makes me feel like a hacker so i do that instead.
+
+Type this text into the command line:
+
+`git clone <copied link>`
+
+It should look something like this:
+
+`git clone https://github.com/SciBorgs/SciGuides.git`
+
+This would clone this repository, the SciBorgs' SciGuides onto your computer in your working directory.
+
+### Branching
```

Comment:
For all of these operation examples, you might want to link to the official git documentation for really in-depth and specific options and information that readers can access. 

It might be a bit overwhelming though... not too sure. If its complex nature is specified it'll work, but I don't know many people that would want to look at it anyway. Google's enough, probably. Thoughts?

Reply:
done